README.md

```diff
@@ -1 +1,28 @@
-# Review_project
\ No newline at end of file
+# Review_project
```

Comment:
Hello, Alonsofl Please I think I gave an inline comment as requested by the first reviewer. I would appreciate it if you could check the review again because I initially used the GitHub inline review as suggested by the first reviewer.  Kind Regards

Reply:
Hello, @SirriRyisa! I would suggest checking my review/comment and also the previous reviewer comment and you will see that these look different from yours.

  Comment:
  ![image](https://user-images.githubusercontent.com/81522435/144931807-075ba8ad-c046-4289-ab47-a8f9973b3afe.png)

  Inline comment (below the review summary):
  ![image](https://user-images.githubusercontent.com/81522435/144931992-4ea07aa6-5b65-4757-80dc-e06167fcebd7.png)

Reply:
I have added the inline comment on the review project